airbyte-webapp/src/views/Connection/ConnectionForm/ScheduleField/ScheduleField.tsx

```diff
@@ -128,6 +129,10 @@ const ScheduleField: React.FC = () => {
     return form.values.scheduleType === ConnectionScheduleType.cron;
   };
 
+  const cronValidationError = useMemo(() => {
+    return (errors?.scheduleData as ConnectionScheduleData)?.cron?.cronExpression;
+  }, [errors]);
```

Comment:
If we want to get rid of the typecasting here (which imho looks a bit weird, given that TS thinks `scheduleData` is `string` here, we could directly pull the error of that field (and get rid of the whole `useMemo` here):

```ts
  const [, { error }] = useField("scheduleData.cron.cronExpression");
```

Reply:
Nice, that is cleaner, I'll use that. It still is not super type safe (`useField()` will accept any string without complaining) but it's not as ugly 😅 

I think to get proper typing we would need to change the yup schema away from overloading `scheduleData` with the ambiguous `.mixed().when(...)`, but I didn't want to blow up the PR.